include/opendht/indexation/pht.h

```diff
@@ -166,6 +175,125 @@ class Pht {
     void insert(Key k, Value v, Dht::DoneCallbackSimple cb = {});
 
 private:
+    class Cache {
+    public:
+        /**
+         * Insert all needed node into the tree according to a prefix
+         * %param p : Prefix that we need to insert
+         */
```

Comment:
Yup i saw that, in fact i use %param on my own doc in C and forget to change it here :)
I'll make an update for that then